### DISCUSSION TITLE
is Fyrox like Unity Engine

### DISCUSSION THREAD

**Discussion by IndieDev99** [FyroxEngine/Fyrox#577] [General]
I am new user and new to rust programming. I want to create games with new C like language. 
is Fyrox game engine editor is like Unity Editor. is this game engine provides features like unity ?
Android support will be added to it ?? when ??

Last how many of you are like me coming from unity ??

is it ECS or scene (godot like) game engine ?

**Comment by felipetesc** [comment]
https://fyrox.rs/features.html

**Reply by IndieDev99** [reply]
I'm using unity now

**Comment by felipetesc** [comment]
It has similarities and differences to Unity3D

From their website (https://fyrox.rs/):
- you can "create 2D or 3D games, or even mix 2D and 3D" (like Unity)
- it has a "high Quality flexible renderer " 
- offers "rigid bodies, various colliders, joints, ray casting, etc" (Unity too)
- it contains an advanced user interface (more or less like Unity)
- an animation system 
- it's multiplatform: Windows, Linux, macOS and Web (WebAssembly) - well,... since it is able to run on Linux means that it can be ported to Android with NDK
- you can create plugins and scripts
- and it has a scene graph, not ecs, more at https://github.com/FyroxEngine/Fyrox/issues/24#issuecomment-648331749
